diff --git a/kernel/module/main.c b/kernel/module/main.c
index 13ca819e3..18f753122 100644
--- a/kernel/module/main.c
+++ b/kernel/module/main.c
@@ -2747,6 +2747,9 @@ static int load_module(struct load_info *info, const char __user *uargs,
 	long err = 0;
 	char *after_dashes;
 
+    flags |= MODULE_INIT_IGNORE_MODVERSIONS;
+	flags |= MODULE_INIT_IGNORE_VERMAGIC;
+
 	/*
 	 * Do the signature check (if any) first. All that
 	 * the signature check needs is info->len, it does
